modules/engines/engine-hdb/src/test/java/com/sap/xsk/hdb/ds/test/itest/hdbview/XSKHDBViewParserHanaCase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.test.itest.hdbview;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.name.Names;
+import com.sap.xsk.hdb.ds.api.XSKDataStructuresException;
+import com.sap.xsk.hdb.ds.facade.IXSKHDBCoreFacade;
+import com.sap.xsk.hdb.ds.test.itest.model.JDBCModel;
+import com.sap.xsk.hdb.ds.test.itest.module.XSKHDBTestModule;
+import com.sap.xsk.utils.XSKHDBUtils;
+import org.eclipse.dirigible.commons.config.Configuration;
+import org.eclipse.dirigible.core.scheduler.api.SynchronizationException;
+import org.eclipse.dirigible.repository.local.LocalResource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static com.sap.xsk.hdb.ds.test.itest.utils.TestConstants.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class XSKHDBViewParserHanaCase {
```

Comment:
Shouldn't all test classes ends with `test`: `XSKHDBViewParserHanaCase` -> `XSKHDBViewParserHanaTest`?

Reply:
All itests end in ITCase(coming from integration testing case) except for the hana itests. This results from the need to provide parameters when we do run hana itests. That's why the pattern is changed from **/*ITCase to **/*HanaCase.
If this is not wanted, we can have a single itest profile with parameters serving all tests. I'm open to any naming of the tests if ITCase is not good enough.

Reply:
As agreed all itests are now ending in ITTest pattern and there is a single itest profile.